### PR TITLE
MB-5593 | gosec G402: annotate all remaining g402s

### PR DESCRIPTION
**Pull request by sojeri**
## Description

Add annotations for all remaining G402s. The reason we want to keep all these remaining insecure verify bools is we are only setting them to true during local development / for local testing.

### Useful historical references

- [PR in which the prime API side was added](https://github.com/transcom/mymove/pull/3395)
- [PR in which the webhook client was added](https://github.com/transcom/mymove/pull/4531)
- [Doc link 1 from annotations (official prime API docs)](https://github.com/transcom/prime_api_deliverable/wiki/Getting-Started#run-prime-api-client)
- [Doc link 2 from annotations (transcom/mymove wiki)](https://github.com/transcom/mymove/wiki/How-to-Test-the-Prime-API-(Local,-Staging,-and-Experimental)#testing-locally)

## Reviewer Notes

How is the phrasing? Should we include those doc refs or refs to the above PRs in the annotation?

## Other references

- [Jira story](https://dp3.atlassian.net/browse/MB-5593)

### CODE REVIEW

File: cmd/prime-api-client/utils/connection.go [hunk 1]
```diff
@@ -44,7 +44,17 @@ func CreatePrimeClientWithCACStoreParam(v *viper.Viper, store *pksigner.Store) (
...
+		// * https://github.com/transcom/prime_api_deliverable/wiki/Getting-Started#run-prime-api-client
+		// * https://github.com/transcom/mymove/wiki/How-to-Test-the-Prime-API-(Local,-Staging,-and-Experimental)#testing-locally
+		//RA Developer Status: {RA Request, RA Accepted, POA&M Request, POA&M Accepted, Mitigated, Need Developer Fix, False Positive, Bad Practice}
+		//RA Validator Status: {RA Accepted, Return to Developer, Known Issue, Mitigated, False Positive, Bad Practice}
+		//RA Validator: jneuner@mitre.org
+		//RA Modified Severity:
+		// #nosec G402
 		tlsConfig := &tls.Config{
 			Certificates:       []tls.Certificate{*cert},
 			InsecureSkipVerify: insecure,
```
Comment (lynzt) [inline cmd/prime-api-client/utils/connection.go:60]: where are we setting this value?

**Review by lynzt — COMMENTED** [review]
I see where we're initing this, but not the actual value coming in... Where are we actually passing this in?

File: cmd/prime-api-client/utils/connection.go (hunk 1 above)
Comment (sojeri) [reply cmd/prime-api-client/utils/connection.go:60]: if you refer to the docs linked in the annotation, you'll see it gets passed through as a CLI arg. you can also chase "insecure" and "InsecureFlag" through the codebase if you want to see the whole flow.

- [Doc link 1 from annotations (official prime API docs)](https://github.com/transcom/prime_api_deliverable/wiki/Getting-Started#run-prime-api-client)
- [Doc link 2 from annotations (transcom/mymove wiki)](https://github.com/transcom/mymove/wiki/How-to-Test-the-Prime-API-(Local,-Staging,-and-Experimental)#testing-locally)

Comment (kimallen) [reply cmd/prime-api-client/utils/connection.go:60]: I'm seeing that.  I can approve since Lindsay is out for the day.

**Review by kimallen — APPROVED** [review]
Approved- I see where the flag is passed

**Review by jneuner-mitre — CHANGES_REQUESTED** [review]
Please add the RA Developer Status before merging.  It tremendously helps us to understand the dev position when we drop into the PR.  Otherwise, lgtm!

**Review by jneuner-mitre — APPROVED** [review]
Thanks for fixing up the statuses!

**Pull request merged**